CMakeLists.txt

```diff
@@ -94,6 +94,8 @@ list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}/share/cmake")
 list(APPEND CMAKE_MODULE_PATH "${DYNAWO_HOME}/cmake")
 list(APPEND CMAKE_MODULE_PATH "${DYNAWO_HOME}/share/cmake")
 
+set(BOOST_ROOT ${DYNAWO_HOME})
```

Comment:
To comply with "standard" way to use "var_ROOT" variables, dynawo_home is only the default value o fBoost_root and log a warning or error if value doesn't match dynawo home

Reply:
done